bin/dasht-query-line

```diff
@@ -87,19 +97,21 @@
 # Distributed under the terms of the ISC license (see the LICENSE file).
 
 : ${DASHT_DOCSETS_DIR:=${XDG_DATA_HOME:-$HOME/.local/share}/dasht/docsets}
+: ${DASHT_FRAGMENTS_CONF:=${XDG_DATA_HOME:-$HOME/.local/share}/dasht/fragments.conf}
 
 pattern=$(echo " $1 " | sed -e 's/[%_]/\\&/g' -e 's/[[:space:]]\{1,\}/%/g')
 test $# -gt 0 && shift # shift off PATTERN so argv contains solely DOCSETs
 
 status=44 # (default) exit with a nonzero status when no results are found
 trap 'status=0' USR1 # override default exit status when results are found
+[ -r "$DASHT_FRAGMENTS_CONF" ] && fragments_conf="$DASHT_FRAGMENTS_CONF" || fragments_conf=
```

Comment:
We might try just reading the config file's contents in one shot here, rather than preparing to read it.

```sh
fragments_conf=$(test -s "$DASHT_FRAGMENTS_CONF" && cat "$DASHT_FRAGMENTS_CONF")
```